modules/tensor_mechanics/include/materials/ComputeMultipleInelasticStress.h

```diff
@@ -13,6 +13,7 @@
 #include "ComputeFiniteStrainElasticStress.h"
 
 class StressUpdateBase;
+enum class TangentCalculationMethod;
```

Comment:
We should probably just `#include StressUpdateBase` to avoid re-declaring `TangentCalculationMethod` here.

Reply:
I wasn't sure the best method. I'll change to #include